examples/tabular/example_tabular_predictor_v2.py

```diff
@@ -0,0 +1,62 @@
+
+from autogluon.tabular import TabularPrediction as task
+from autogluon.tabular.task.tabular_prediction.predictor import TabularPredictor
```

Comment:
per discussed, we will rename it `from autogluon.tabular import Predictor`, so other tasks can have a similar import sentence

Reply:
In later meeting we came to the conclusion that calling every task's predictor `Predictor` would be confusing, especially for tasks with multiple predictor variants.

Instead planning to revise name to:

```
from autogluon.tabular import TabularPredictor
predictor = TabularPredictor(...)
```